packages/runner/src/builtins/wish.ts

```diff
@@ -47,14 +46,25 @@ function parseWishTarget(target: string): ParsedWishTarget | undefined {
   if (trimmed.startsWith("#")) {
     const rest = trimmed.slice(1);
     const segments = rest.split("/").filter((segment) => segment.length > 0);
-    const baseSegment = segments.shift();
-    if (!baseSegment) return undefined;
-    const key = `#${baseSegment}`;
-    return { key, path: segments };
+
+    // Check if it's a well-known target like #allCharms
+    if (segments.length > 0) {
+      const baseSegment = segments[0];
+      const key = `#${baseSegment}`;
+      // If it's a known well-known target, use it with remaining path
+      if (key === "#allCharms") {
+        return { key, path: segments.slice(1) };
+      }
```

Comment:
Ultimately we can drop this once we're accustomed to getting `/allCharms` to grab it from the space cell.